.rubocop_todo.yml

```diff
@@ -13,40 +13,33 @@ Lint/AmbiguousRegexpLiteral:
     - 'test/vagrant-service-manager/binary_handlers/adb_openshift_binary_handler_test.rb'
 
 # Offense count: 1
-Lint/HandleExceptions:
+Lint/Eval:
   Exclude:
-    - 'lib/vagrant-service-manager/service.rb'
+    - 'features/support/env.rb'
 
-# Offense count: 13
+# Offense count: 17
```

Comment:
@hferentschik I think this offenses has been accumulated from last we ran `rubocop --auto-gen-config`. I think we need to create an issue to do a first set of rubocop offense cleanup.